Src/Sarif.PatternMatcher/AnalyzeCommand.cs

```diff
@@ -181,26 +224,11 @@ protected override ISet<Skimmer<AnalyzeContext>> CreateSkimmers(AnalyzeOptions o
             return CreateSkimmersFromDefinitionsFiles(this.FileSystem, options.SearchDefinitionsPaths);
         }
 
-        private static string PushData(string text, Dictionary<string, string> sharedStrings)
-        {
-            if (sharedStrings == null || text?.Contains("$") != true)
-            {
-                return text;
-            }
-
-            foreach (string key in sharedStrings.Keys)
-            {
-                text = text.Replace(key, sharedStrings[key]);
-            }
-
-            return text;
-        }
-
         private static void ThrowInvalidSharedStringsEntry(string line)
         {
             throw new InvalidOperationException(
-                "Malformed shared strings entry. Every shared string should consist of a " +
-                "key name (prefixed with $) followed by an equals sign and the string value " +
+                $"Malformed shared strings entry. Every shared string should consist of a " +
```

Comment:
>$ [](start = 16, length = 1)

nit: remove if not needed.

Reply:
This is for style, it makes all the string literals line up exactly. Does it bother you? :)

---
In reply to: [555679916](https://github.com/microsoft/sarif-pattern-matcher/pull/101#discussion_r555679916) [](ancestors = 555679916)